doc/source/examples/multiphysics/air-bubble-compression/air-bubble-compression.rst

```diff
@@ -0,0 +1,337 @@
+---------------------------
+
+- Parameter file: ``examples/multiphysics/air-bubble-compression/air-bubble-compression.prm``
+- Python script for postprocessing: ``examples/multiphysics/air-bubble-compression/air-bubble-compression-postprocessing.py``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+A circular air bubble of diameter :math:`D=0.06` lies at the center of a square-shaped domain with sides of length :math:`L=0.1`.
+On all four sides of the domain, water penetrates with a constant velocity norm of :math:`||\mathbf{v}||=0.0025` causing the compression of the air bubble.
+The initial configuration of this example is illustrated below.
+
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/air-bubble-initial-configuration.svg                                                          |
+|     :align: center                                                                                                |
+|     :width: 800                                                                                                   |
+|     :name: Air bubble initial configuration                                                                       |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+  In this example, gravity and surface tension forces are not considered.
+
+--------------
+Parameter File
+--------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+Time integration is handled by a 2nd-order backward differentiation scheme (``bdf2``) with a variable time step.
+The initial time step is set to :math:`0.1 \, \text{s}` and the simulation lasts :math:`2.5 \, \text{s}`.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method           = bdf2
+      set time end         = 2.5
+      set time step        = 0.1
+      set adapt            = true
+      set max cfl          = 1
+      set output name      = air-bubble-compression
+      set output frequency = 5
+      set output path      = ./output/
+    end
+
+Multiphysics
+~~~~~~~~~~~~
+
+The ``multiphysics`` subsection is used to enable the VOF solver.
+
+.. code-block:: text
+
+    subsection multiphysics
+      set VOF  = true
+    end 
+
+VOF
+~~~
+
+In the ``VOF`` subsection, the ``interface sharpening`` and the ``phase filtration`` features are enabled.
+The interface sharpening method and its parameters are explained in the :doc:`../dam-break/dam-break` example.
+The phase filtration filters the phase field used for the calculation of physical properties by stiffening the value of the phase fraction.
+We refer the reader to the :doc:`../../../../parameters/cfd/volume_of_fluid` documentation for further explanation on the phase filtration.
```

Comment:
They are now described in the vof theory guide :)

Reply:
You're right I'll change it in the rising bubble example also :)